packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1113,6 +1113,24 @@ class TextTreeRenderer {
     if (kReleaseMode) {
       return '';
     }
+    String result;
```

Comment:
is there a reason this needs to be stripped out in profile mode as well as release mode?
For release mode this is set to short circuit to the empty string so probably also have it short circuit to that in profile mode if you want it to short circuit there.

Reply:
I replied below. If you think this specific one is valuable in profile mode, i can revert.

Reply:
this one will shutdown all display of FlutterError objects in profile mode. @goderbauer what are your thoughts on that? Errors are by default displayed using debugPrint which appears to still execute outside of debug mode. I don't have the context to know whether having debugPrint still execute in profile+release mode is an oversight or by design.

Reply:
It seems valuable to me to still dump errors to the console in profile builds as its often used to debug (performance) issues and just silently swallowing errors there seems like a cause for frustration.

Reply:
Ok i reverted this code by placing into else clause so release mode it is removed but kept in profile.